x-pack/plugins/enterprise_search/public/applications/ai_search/components/ai_search_guide/elser_panel.tsx

```diff
@@ -7,125 +7,71 @@
-          />
-        </p>
-      </EuiText>
-    ),
-    status: 'incomplete',
-  },
-];
+export const ElserPanel: React.FC = () => {
+  const { application } = useValues(KibanaLogic);
 
-export const ElserPanel: React.FC = () => (
-  <>
-    <EuiSpacer />
-    <EuiFlexGroup direction="column">
-      <EuiFlexItem grow={false}>
-        <EuiText>
-          <p>
-            <FormattedMessage
-              id="xpack.enterpriseSearch.aiSearch.elser.description"
-              defaultMessage="Effortlessly deploy the {elser} for instant text semantic search capabilities in just a few clicks. This model expands your document and query text using the 'text_expansion' field, delivering seamless search out of the box."
-              values={{
-                elser: (
-                  <EuiLink target="_blank" href={docLinks.elser} external={false}>
-                    {i18n.translate(
-                      'xpack.enterpriseSearch.aiSearch.elser.description.elserLinkText',
-                      {
-                        defaultMessage: 'Elastic Learned Sparse Encoder v2',
-                      }
-                    )}
-                  </EuiLink>
-                ),
-              }}
-            />
-          </p>
-        </EuiText>
-      </EuiFlexItem>
-      <EuiFlexItem grow={false}>
-        <EuiSteps steps={steps} titleSize="xs" />
-      </EuiFlexItem>
-    </EuiFlexGroup>
-  </>
-);
+  return (
+    <>
+      <EuiSpacer />
+      <EuiFlexGroup direction="column">
+        <EuiFlexItem grow={false}>
+          <EuiText>
+            <p>
+              <FormattedMessage
+                id="xpack.enterpriseSearch.aiSearch.elser.description"
+                defaultMessage="The {elser} enables textual semantic search over your Elasticsearch documents in just a few commands. Use the 'semantic_text' field to simplify model deployment and generate sparse vector document embeddings."
+                values={{
+                  elser: (
+                    <EuiLink
+                      data-test-subj="enterpriseSearchElserPanelElasticLearnedSparseEncoderV2Link"
+                      target="_blank"
+                      href={docLinks.elser}
+                      external={false}
+                    >
+                      {i18n.translate(
+                        'xpack.enterpriseSearch.aiSearch.elser.description.elserLinkText',
+                        {
+                          defaultMessage: 'Elastic Learned Sparse Encoder v2',
+                        }
+                      )}
+                    </EuiLink>
+                  ),
+                }}
+              />
+            </p>
+          </EuiText>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiLink
+            data-test-subj="enterpriseSearchElserPanelLink"
+            onClick={() =>
+              application.navigateToApp(
+                SEMANTIC_SEARCH_PLUGIN.URL.replace(/^(?:\/app\/)?(.*)$/, '$1') +
```

Comment:
You don't need to do this, you can either use `application.navigateToUrl()` without the regex replace to strip `/app`. or you could build an href link using `http` from the KibanaLogic. Something like

```
 href={http.basePath.prepend(
  `${SEMANTIC_SEARCH_PLUGIN.URL}?model_example=elser`
)}
```